docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
+
+With Inago, it's a piece of cake to deploy an Elasticsearch cluster,
+as well as updating it without any downtime or data loss.
+
+It should be noted that this is a demo, and there are dangers to using it in a production environment without further testing.
+Saying that, if you want to do so, and have questions, the team is always available on IRC (`#giantswarm` on `freenode`) <3
+
+## Setup
+- We're using a CoreOS cluster running locally, with Vagrant. See https://coreos.com/os/docs/latest/booting-on-vagrant.html for further information.
+- There are three nodes in the cluster, and each node has 2048MB of memory allocated to it.
+- The `inago` binary, as well as the `elasticsearch` directory from `example`, have been copied to all the nodes in the cluster.
+
+## Starting the Elasticsearch cluster
+We're going to use Inago's `up` command to bring up multiple group slices of the `elasticsearch` group. `up` is equivalent to `submit`, followed by `start`.
+
+Due to the `Conflicts` statement in the unit file for the group, each group slice will be automatically scheduled onto separate machines.
+```
+core@core-01 ~ $ ./inagoctl up elasticsearch 3
+2016-04-06 16:57:39.644 | INFO     | context.Background: Succeeded to submit group 'elasticsearch'.
+2016-04-06 16:57:46.656 | INFO     | context.Background: Succeeded to start 3 slices for group 'elasticsearch': [1f4 8f1 9e9].
+```
+As you can see, the `elasticsearch` group has been submitted and started. Each group slice has an ID, which can be seen in the output.
+
+We can use the `status` command to inspect the state of the group. The _verbose_ flag is used so that the hash of the unit file is also printed. This will be interesting later, after we upgrade Elasticsearch. You can also see the the group slices have been scheduled onto multiple machines.
+```
+core@core-01 ~ $ ./inagoctl status -v elasticsearch
+Group              Units                      FDState   FCState   SAState  Hash                                      IP            Machine
+
+elasticsearch@2cf  elasticsearch@2cf.service  launched  launched  active   07969bf8caa1a7f8c73b35bb826229425d3264f6  172.17.8.103  1564e0d3c4ab4aadbe76b24cea90c62f
+elasticsearch@445  elasticsearch@445.service  launched  launched  active   07969bf8caa1a7f8c73b35bb826229425d3264f6  172.17.8.102  2dbbe125e237410bb946e7b0f6b95e4c
+elasticsearch@891  elasticsearch@891.service  launched  launched  active   07969bf8caa1a7f8c73b35bb826229425d3264f6  172.17.8.101  b89277d82f6a425eb83abb39c00485bc
+```
+
+We can also check Elasticsearch via its own API.
```

Comment:
Do we want to note that the endpoint in the curl request needs to be modified according to the personal setup?